library-core/src/main/java/com/mikepenz/fastadapter/FastAdapter.kt

```diff
@@ -55,11 +55,14 @@ open class FastAdapter<Item : GenericItem> : RecyclerView.Adapter<RecyclerView.V
     // the total size
     private var globalSize = 0
 
+    private var _eventHooks: MutableList<EventHook<out Item>>? = null
+
     /**
      * The eventHooks handled by this FastAdapter
      */
-    var eventHooks: MutableList<EventHook<out Item>>? = null
-        private set
+    val eventHooks: MutableList<EventHook<out Item>>
```

Comment:
This changes the api, so if you don't want this I can remove it.

Reply:
Should be fine so far. Chances of someone overwriting this one particular should be rather low